src/bin/sol-fbp-runner/runner.c

```diff
@@ -146,28 +146,47 @@ add_simulation_node(struct runner *r, const char *node_type, const char *port_na
 }
 
 struct map {
-    const char *packet_name;
+    const struct sol_flow_packet_type *packet;
     const char *node_type;
     const char *port_name;
 };
 
-static const struct map input_nodes[] = {
-    { "IRange", "gtk/spinbutton", "OUT" },
-    { "DRange", "gtk/slider", "OUT" },
-    { "Any", "gtk/toggle", "OUT" },
-    { "Empty", "gtk/toggle", "OUT" },
-    { "Boolean", "gtk/pushbutton", "OUT" },
-    { "RGB", "gtk/rgb-editor", "OUT" },
-    { "Byte", "byte/editor", "OUT" },
+static struct map input_nodes[] = {
+    { NULL, "gtk/spinbutton", "OUT" },
+    { NULL, "gtk/slider", "OUT" },
+    { NULL, "gtk/toggle", "OUT" },
+    { NULL, "gtk/toggle", "OUT" },
+    { NULL, "gtk/pushbutton", "OUT" },
+    { NULL, "gtk/rgb-editor", "OUT" },
+    { NULL, "byte/editor", "OUT" },
 };
 
-static const struct map output_nodes[] = {
-    { "IRange", "gtk/label", "IN" },
-    { "DRange", "gtk/label", "IN" },
-    { "String", "gtk/label", "IN" },
-    { "Boolean", "gtk/led", "IN" },
+static struct map output_nodes[] = {
+    { NULL, "gtk/label", "IN" },
+    { NULL, "gtk/label", "IN" },
+    { NULL, "gtk/label", "IN" },
+    { NULL, "gtk/led", "IN" },
 };
 
+static void
+init_tables() {
```

Comment:
Functions without parameters in C should have a `"void"` argument: `init_tables(void)`.